core/scheduler/build.gradle

```diff
@@ -66,6 +66,9 @@ dependencies {
 
     compile "org.scala-lang:scala-library:${gradle.scala.version}"
     compile "io.altoo:akka-kryo-serialization_${gradle.scala.depVersion}:1.0.0"
+    compile "com.lightbend.akka.management:akka-management-cluster-http_${gradle.scala.depVersion}:${gradle.akka_management.version}" 
```

Comment:
IIRC, our downstream is not using these libraries while we are running OW on Kubernetes.
@ningyougang Do you have any idea about the difference between downstream and upstream?

Reply:
If there's a different way to do this, I am happy to hear it and chance it if need be!

Reply:
@style95 , have no idea why we don't add this 
```
compile "com.lightbend.akka.management:akka-management-cluster-http_${gradle.scala.depVersion}:${gradle.akka_management.version}"
```
@hunhoffe , why do you add this? which code would use this lib?

Reply:
@ningyougang thank you, you are correct, akka-management-cluster-http is not needed although the other two are. I've removed in recent commit.